app/script/conversation/ConversationMapper.js

```diff
@@ -22,21 +22,53 @@
 'use strict';
 
 /**
- * @typedef {object} SelfStatusUpdate
- * @property {number=} archived_timestamp
- * @property {number=} cleared_timestamp
- * @property {number=} ephemeral_timer
- * @property {number=} message_timer
- * @property {number=} last_event_timestamp
- * @property {number=} last_read_timestamp
- * @property {number=} last_server_timestamp
- * @property {boolean=} otr_archived
+ * @typedef {object} ConversationData
```

Comment:
Shouldn't we precise if it is data from the backend or the db?

Reply:
#changed